Source/Core/Core/PowerPC/JitLLVM/JitLLVM_Tables.h

```diff
@@ -0,0 +1,15 @@
+// Copyright 2015 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Core/PowerPC/Gekko.h"
+#include "Core/PowerPC/PPCTables.h"
+#include "Core/PowerPC/JitLLVM/Jit.h"
```

Comment:
Forward declarations help increase compilation speed (preprocessor doesn't need to do it's regular copy/paste into the file, etc). Feel free to ignore it if you want.